src/neo/Ledger/Blockchain.cs

```diff
@@ -65,14 +65,11 @@ private class UnverifiedBlocksList { public LinkedList<Block> Blocks = new Linke
         private readonly ConcurrentDictionary<UInt256, Block> block_cache = new ConcurrentDictionary<UInt256, Block>();
         private readonly Dictionary<uint, UnverifiedBlocksList> block_cache_unverified = new Dictionary<uint, UnverifiedBlocksList>();
         internal readonly RelayCache RelayCache = new RelayCache(100);
-        private SnapshotCache currentSnapshot;
         private ImmutableHashSet<UInt160> extensibleWitnessWhiteList;
 
         public IStore Store { get; }
         public DataCache View => new SnapshotCache(Store);
```

Comment:
Because it returns a `IDisposable` object, I think that we should move this to a method instead of a property, like `GetView` this will remember us to use `using` after calling it.

Reply:
The magic is that this object does not actually need to be disposed. Because the `ISnapshot` inside it is `null`.